sopel/modules/url.py

```diff
@@ -154,7 +149,7 @@ def shutdown(bot):
 @plugin.example('.urlpexclude example\\.com/\\w+', user_help=True)
 @plugin.example('.urlexclude example.com/path', user_help=True)
 @plugin.output_prefix('[url] ')
-def url_ban(bot, trigger):
+def url_ban(bot: Sopel, trigger: Trigger):
```

Comment:
This is not an instance of `Sopel` but of `SopelWrapper`, true for the other commands & rules.

Reply:
Ah, thanks. IIRC in the safety PR you said to use either - same there? Or what's different?

Reply:
Plugin callables need a SopelWrapper. Job, Setup, Teardown: Sopel. Everything else: it depends.